apis/v1beta1/httproute_types.go

```diff
@@ -156,10 +156,13 @@ type HTTPRouteRule struct {
 	// ties. Across all rules specified on applicable Routes, precedence must be
 	// given to the match with the largest number of:
 	//
-	// * Characters in a matching path.
+	// * Characters in a matching "Exact" path match
+	// * Characters in a matching "Prefix" path match
 	// * Header matches.
 	// * Query param matches.
 	//
+	// Note: The precedence of RegularExpression path matches are implementation-specific.
```

Comment:
I agree with the decision to make precedence for RE implementation-specific, as I think we would likely have created a conundrum for several implementations.